superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberPeriodOverPeriod/controlPanel.ts

```diff
@@ -180,14 +108,12 @@ const config: ControlPanelConfig = {
         ],
       ],
     },
+    sections.timeComparisonControls(false, false, false),
```

Comment:
I'd suggest passing an object here instead of arguments so that we can tell which values are being set to false here for example.